.github/workflows/install-ssf.yaml

```diff
@@ -14,28 +17,52 @@ jobs:
     runs-on: ubuntu-latest
     name: Test SSF Installation
     steps:
-    - uses: actions/checkout@v2
-    - name: Start minikube
-      run: |
-        make setup-minikube
-    - name: Try the cluster !
-      run: kubectl get pods -A
-    - name: Deploy SSF to minikube
-      run: |
-        make setup-tekton-chains
-    - name: Generate temp keys
-      run: |
-        make tekton-generate-keys
-    - name: Setup Kyverno
-      run: |
-        make setup-kyverno
-    - name: Run test pipeline
-      run: |
-        ./examples/buildpacks/buildpacks.sh
-        export DOCKER_IMG=$(tkn pr describe --last -o jsonpath='{.spec.params[?(@.name=="APP_IMAGE")].value}')
-        tkn pr logs --last -f
-        sleep 60
-        docker run --rm gcr.io/go-containerregistry/crane ls $DOCKER_IMG
-        tkn tr describe --last -o json | jq -r '.metadata.annotations["chains.tekton.dev/signed"]'
-        cosign verify --key k8s://tekton-chains/signing-secrets ${DOCKER_IMG}
-        cosign verify-attestation --key k8s://tekton-chains/signing-secrets ${DOCKER_IMG}
+      - uses: actions/checkout@v2
+      - name: Start minikube
+        run: |
+          make setup-minikube
+      - name: Try the cluster !
+        run: kubectl get pods -A
+      - name: Deploy SSF to minikube
+        run: |
+          make setup-tekton-chains
+      - name: Generate temp keys
+        run: |
+          make tekton-generate-keys
+      - name: Setup Kyverno
+        run: |
+          make setup-kyverno
+      - name: Run test pipeline
+        run: |
+          ./examples/buildpacks/buildpacks.sh
+          export DOCKER_IMG=$(tkn pr describe --last -o jsonpath='{.spec.params[?(@.name=="APP_IMAGE")].value}')
+          tkn pr logs --last -f
+          sleep 60
+          docker run --rm gcr.io/go-containerregistry/crane ls $DOCKER_IMG
+          tkn tr describe --last -o json | jq -r '.metadata.annotations["chains.tekton.dev/signed"]'
+          cosign verify --key k8s://tekton-chains/signing-secrets ${DOCKER_IMG}
+          cosign verify-attestation --key k8s://tekton-chains/signing-secrets ${DOCKER_IMG}
+---
+- uses: actions/checkout@v2
+- name: Start minikube
+  run: |
+    make setup-minikube
+- name: Try the cluster !
+  run: kubectl get pods -A
+- name: Deploy SSF to minikube
+  run: |
+    make setup-tekton-chains
+    make setup-kyverno
+- name: Generate temp keys
+  run: |
+    make tekton-generate-keys
+- name: Run test pipeline
+  run: |
+    ./examples/buildpacks/buildpacks.sh
+    export DOCKER_IMG=$(tkn pr describe --last -o jsonpath='{.spec.params[?(@.name=="APP_IMAGE")].value}')
+    tkn pr logs --last -f
+    sleep 60
+    docker run --rm gcr.io/go-containerregistry/crane ls $DOCKER_IMG
+    tkn tr describe --last -o json | jq -r '.metadata.annotations["chains.tekton.dev/signed"]'
+    cosign verify --key k8s://tekton-chains/signing-secrets ${DOCKER_IMG}
+    cosign verify-attestation --key k8s://tekton-chains/signing-secrets ${DOCKER_IMG}
```

Comment:
This seems to be from an older revision of this file before `make setup-kyverno` was split out into its own step.